plugins/rake-fast/rake-fast.plugin.zsh

```diff
@@ -8,7 +8,11 @@ _rake_refresh () {
 }
 
 _rake_does_task_list_need_generating () {
-  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]]
+  [[ ! -f .rake_tasks ]] || [[ Rakefile -nt .rake_tasks ]] || (_is_rails_app && [[ lib/tasks -nt .rake_tasks ]])
```

Comment:
Any `Rails` project fit. For example you can take [this](https://github.com/ivanovaleksey/confucius) my simple draft project. As I understand all you need is ~~love~~ `lib/tasks` folder.